pom.xml

```diff
@@ -573,7 +573,7 @@
         <commons.logging.version>1.1.1</commons.logging.version>
         <commons.lang3.version>3.3.2</commons.lang3.version>
         <commons.io.version>2.4</commons.io.version>
-        <axis2.version>1.6.2.wso2v13</axis2.version>
+        <axis2.version>1.6.1-wso2v24</axis2.version>
```

Comment:
🤔

Reply:
1.6.2.wso2v13 seems to be due to some mistake. See [1]

[1] http://maven.wso2.org/nexus/content/repositories/releases/org/apache/axis2/wso2/axis2/